### PR TITLE
new acceptance test for a new address

### PR DESCRIPTION
**Pull request by hbrace**
# Motivation and Context

Check to see if a case is created from a NEW_ADDRESS_REPORTED event.
# What has changed

A test has been added to send a new address event and then check case is created and events are logged.
# How to test?

Run tests with other PRs on the trello ticket.
# Links

https://trello.com/c/qixf5J2X/594-am1-part-1-new-address-reported-event-13
# Screenshots (if appropriate):

### CODE REVIEW

**Comment by hbrace** [comment]
[AM1 part 1 - New Address Reported event (13)](https://trello.com/c/qixf5J2X/594-am1-part-1-new-address-reported-event-13)

**Review by LukeLoze1 — APPROVED** [review]

**Review by NickGrantX — CHANGES_REQUESTED** [review]
I think we need a test for creating a _skeleton_ case too.

File: acceptance_tests/features/steps/address.py [hunk 1]
```diff
@@ -37,6 +39,112 @@ def check_case_events(context):
...
+            message=message,
+            content_type='application/json',
+            routing_key=Config.RABBITMQ_ADDRESS_ROUTING_KEY)
+
+
+@step('the case can be retrieved')
+def retrieve_case(context):
+    response = requests.get(f'{caseapi_url}{context.case_id}?caseEvents=true')
+    test_helper.assertEqual(response.status_code, 200, 'Case not found')
+    context.first_case = response.json()
```
Comment (NickGrantX) [inline acceptance_tests/features/steps/address.py]: I don't see anywhere that the actual values of the case are checked - they could be wrong

**Review by NickGrantX — CHANGES_REQUESTED** [review]
The acceptance test should check _some_ (ideally all) of the case fields, to make sure that the behaviour is correct and the data has been copied correctly.

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Pull request merged**